website_sale_recently_viewed_products/README.rst

```diff
@@ -0,0 +1,50 @@
+========================
```

Comment:
The title is already put without this line. See OCA's template: https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst